File: Data Structures & Algorithms/duplicate-integer/submission-0.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums: number[]): boolean {
        const dup: any = {}

        for (let num of nums) {
            if (dup[num] >= 1) {
               return true
            }
            dup[num] = 1
        }
        return false
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        unique = set(nums)
        longest = 0
        for num in nums:
            if (num - 1) not in unique:
                length = 0
                while (num + length in unique):
                    length += 1
                longest = max(length, longest)

        return longest
